build/index.html

```diff
@@ -1,20 +1,27 @@
-<html lang="en">
- <head>
-   <meta charset="utf-8">
-   <meta name="viewport" content="width=device-width,initial-scale=1">
-   <meta http-equiv="x-ua-compatible" content="ie=edge">
-   <link rel="shortcut icon" href="../assets/images/favicon.png">
-   <title>STAC API</title>
- </head>
- <body>
-   <div id="content"></div>
-   <script src="https://cdn.jsdelivr.net/npm/redoc@next/bundles/redoc.standalone.js"> </script>
-   <script type="text/javascript">
-     Redoc.init('openapi.yaml', {
-       expandResponses:"200,201,202,203,204",
```

Comment:
Is it intentional that you drop those settings?

Reply:
I just readded them. I'm not sure why we made these changes to the default, but I also didn't test them to see how they were different. I'll try that out before changing.

Reply:
I made them, they give a bit better UI experience IMHO.